baseapp_auth/rest_framework/users/tasks.py

```
@@ -0,0 +1,42 @@
+                participant_qs = ChatRoomParticipant.objects.filter(profile__user=user)
+                room_ids = list(participant_qs.values_list("room_id", flat=True).distinct())
+                participant_qs.delete()
+
+                for room_id in room_ids:
+                    room = ChatRoom.objects.get(id=room_id)
+                    room.participants_count = ChatRoomParticipant.objects.filter(room=room).count()
+                    room.save(update_fields=["participants_count"])
+            user.delete()
+        send_anonymize_user_success_email(user_email)
+    except Exception as e:
+        logging.exception(e)
```

Ruff (0.14.0)

Redundant exception object included in `logging.exception` call

(TRY401)